### PR TITLE
Added Expanded boolean flag 

### PR DESCRIPTION
**Pull request by Pradeepkn**
Right aligning drop down image was not happening and on enabling expanded flag it works.
Added isExpandEnabled flag

### CODE REVIEW

File: lib/country_picker_dropdown.dart [hunk 1]
```diff
@@ -11,6 +11,7 @@ class CountryPickerDropdown extends StatefulWidget {
     this.itemBuilder,
     this.initialValue,
     this.onValuePicked,
+    this.isExpandEnabled
```
Comment (benz93chung) [inline lib/country_picker_dropdown.dart]: I think calling it isExpanded should be enough

Comment (Pradeep-ShareChat) [reply lib/country_picker_dropdown.dart]: Updated the flag name.

File: lib/country_picker_dropdown.dart [hunk 2]
```diff
@@ -11,7 +11,7 @@ class CountryPickerDropdown extends StatefulWidget {
     this.itemBuilder,
     this.initialValue,
     this.onValuePicked,
-    this.isExpandEnabled
+    this.isExpanded
```
Comment (figengungor) [inline lib/country_picker_dropdown.dart]: Add false as default value to isExpanded, please.

**Comment by figengungor** [comment]
Thank you=)

**Comment by Pradeep-ShareChat** [comment]
Please let me know when will be the new version published?

**Pull request merged**